x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/get_trends.ts

```diff
@@ -27,8 +27,20 @@ export interface ScoreTrendDoc {
 
 export const getTrendsQuery = () => ({
   index: BENCHMARK_SCORE_INDEX_DEFAULT_NS,
-  size: 5,
+  size: 99,
```

Comment:
it was decided that for now we'll bring all the data that we have in the last 24 hours and display everything.